src/main/scala/firrtl/ir/IR.scala

```diff
@@ -119,7 +119,7 @@ case class FixedLiteral(value: BigInt, width: Width, point: Width) extends Liter
   def tpe = FixedType(width, point)
   def serialize = {
     val pstring = if(point == UnknownWidth) "" else s"<${point.serialize}>"
-    s"Fixed${width.serialize}$pstring(" + Utils.serialize(value) + ")"
+    s"""Fixed${width.serialize}$pstring("h""" + value.toString(16) + """")"""
```

Comment:
You could insert these values with `${value.toString(16)}` to make it more consistent